src/mgmt/srv_internal.h

```diff
@@ -70,7 +70,8 @@ int ds_mgmt_create_pool(uuid_t pool_uuid, const char *group, char *tgt_dev,
 int ds_mgmt_destroy_pool(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
 			 const char *group, uint32_t force);
 int ds_mgmt_evict_pool(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
-		       uuid_t *handles, size_t n_handles, const char *group);
+		       uuid_t *handles, size_t n_handles, char* machine,
```

Comment:
(style)  "foo* bar" should be "foo *bar"

Reply:
fixed